### ISSUE TITLE
struct.decode raising "TypeError: Cannot convert undefined or null to object"

### ISSUE THREAD

**Issue by fontelemos** [callmehiphop/pb-util#13]
Hello! I've been having problems with the `struct.decode` method, here's what i did:

I used the [parseDict](https://googleapis.dev/python/protobuf/latest/google/protobuf/json_format.html#google.protobuf.json_format.ParseDict) from `google.protobuf.json_format`(python 3.8) to convert the following payload to a protobuf `Struct`:

```python
{'channel': 'desktop', 'context': {}}
```
The payload is received by a nodejs app which uses the `struct.decode` method, however it raises an error:

Received Payload
```js
{
  fields: { 
      context: { 
          structValue: {}
      },
      channel: { 
          stringValue: 'desktop' 
      }
   }
}
```
Exception
```sh
TypeError: Cannot convert undefined or null to object
```

I've analysed the `pb-util` code and found out that, using this payload, there's a moment when the [variable `fields`](https://github.com/callmehiphop/pb-util/blob/f7f11bf75a9a6c7bdb94a8f17048da39f245eb0e/index.ts#L180) inside the struct.decode is `undefined`, resulting in the exception.

**Comment by callmehiphop** [comment]
hey @Leinvedan, thanks for reporting the issue. I just opened a PR to fix this.

**Comment by callmehiphop** [comment]
Just published v1.0.2, please let me know if the issue persists!

**Comment by fontelemos** [comment]
Just tested here and it's working perfectly! 
Thanks @callmehiphop !!!!
